docker/gogs/app.ini

```
@@ -0,0 +1,54 @@
+BRAND_NAME = Conduit Suite Git Repos
+RUN_USER   = git
+RUN_MODE   = prod
+
+[database]
+TYPE     = sqlite3
+HOST     = 127.0.0.1:5432
+NAME     = gogs
+SCHEMA   = public
+USER     = gogs
+PASSWORD = 
+SSL_MODE = disable
+PATH     = /data/gogs.db
+
+[repository]
+ROOT           = /data/git/gogs-repositories
+DEFAULT_BRANCH = master
```

```suggestion
DEFAULT_BRANCH = main
```

Let's use `main` for the default branch name, and remove the `master` branch.